pkgs/development/python-modules/envisage/default.nix

```diff
@@ -15,7 +15,7 @@ buildPythonPackage rec {
     sha256 = "0zrxlq4v3091727vf10ngc8418sp26raxa8q83i4h0sydfkh2dic";
   };
 
-  propagatedBuildInputs = [ traits apptools ];
+  propagatedBuildInputs = [ traits apptools setuptools ];
```

Comment:
A `pythonImportsCheck` would have caught this, do you mind adding one?

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.

Reply:
Actually in this particular case it doesn't seem to - note below that the tests are enabled for this package, and those pass too.

I think this is probably passing because of the presence of `setuptools` in the build environment.

Reply:
you're right, I could have sworn it used to though.... :(

Reply:
I've filed a Nixpkgs bug (#133032) for reference -- might look into it next week or so if I get a chance.